trunk/examples/programs/regression/c/preprocessor/preprocessor-conditionals_unsafe.c

```diff
@@ -0,0 +1,34 @@
+//#Unafe
```

Comment:
//#Unsafe

Reply:
Thanks for pointing out the missing program correctness specifier.